ETC.tsv

```diff
@@ -14490,11 +14490,11 @@ ETC_20150918_014491	$The fake Tree Root Crystal reduced your movement speed!
 ETC_20150918_014492	$The fake Tree Root Crystal froze you!
 ETC_20150918_014493	They betrayed me!
 ETC_20150918_014494	Looking through the piles of the leaves
-ETC_20150918_014495	The spirit is attacking!
-ETC_20150918_014496	The spirit of the pilgrim that was trapped inside the monster came out!{nl}Release the spirit!
-ETC_20150918_014497	Lure the monster and attack it to pull out the spirit!
-ETC_20150918_014498	To calm the restrained spirit, you would need a flower
-ETC_20150918_014499	The spirit of the pilgrim is attacking the Tree Root Crystal!
+ETC_20150918_014495	The soul is attacking!
+ETC_20150918_014496	The soul of the pilgrim that was trapped inside the monster came out!{nl}Release the soul!
+ETC_20150918_014497	Lure the monster and attack it to pull out the soul!
+ETC_20150918_014498	To calm the restrained soul, you would need a flower
+ETC_20150918_014499	The soul of the pilgrim is attacking the Tree Root Crystal!
```

Comment:
The soul of the pilgrim is attacking the Tree Root Crystal!

to

The pilgrim's soul is attacking the Tree Root Crystal!